### DISCUSSION TITLE
question

### DISCUSSION THREAD

**Discussion by sk9029** [kannagi0303/yt-dlp-gui#108] [Q&A]
Hi, I have been using your site for a long time. I am an amateur who just started using it, but I have three questions.

1yt-dlp-gui can analyze and download videos from instagram, abema.tv, twitch and other websites, but it cannot analyze or download youtube videos.
How can I download youtube videos?

2I can watch the downloaded video files of instagram, abema.tv, twitch, etc. as they are, but most of the video files are not perfect for me unless I encode them. Is this a specification?

3 If you are able to analyze and download on youtube
Is it possible to analyze and download youtube playlists in bulk?

I know this is hard to understand, but I would appreciate your answer.

**Comment by kannagi0303** [comment]
Answer 1: yt-dlp can download from these websites, but currently yt-dlp-gui mainly focuses on YouTube and has lower support for other websites.

Answer 2: In addition to limitations on resolution and length for each website, your playback environment is also crucial, and I cannot provide a solution for that.

Answer 3: Currently, the videos are mainly from YouTube, and playlist support is not yet available (if you have been using my website for a long time, you will notice that this issue has existed for quite some time).

**Reply by sk9029** [reply]
Thank you for your response.

**Reply by sk9029** [reply]
What is your solution?  
Usage environment
Windows 11 Pro  64-bit
yt-dlp-gui Latest version 2023.03.28 
twitch.tv, Instagram, etc. Can be analyzed and downloaded, youtube cannot.

https://user-images.githubusercontent.com/130432931/232689582-546d27e0-14f3-4559-86d9-39088a9e0133.mp4

**Reply by kannagi0303** [reply]
First of all, the URL you used in your video is https://www.youtube.com/watch?v=-ROUYHS8A_A.

I am unable to access this URL properly, as the video may have been removed or there may be age and login restrictions on some videos that require the use of cookies.

If you are familiar with command-line mode, you can use yt-dlp to directly download this video, which will provide you with complete information.

**Reply by sk9029** [reply]
Thanks for the answer.
